lib/logitech_receiver/settings_templates.py

```diff
@@ -408,15 +402,17 @@ def __init__(self, device):
 
             def setNewDpi(self, newDpiIdx):
                 newDpi = self.dpiChoices[newDpiIdx]
-                self.dpiSetting.write(newDpi)
+                if self.dpiSetting:
```

Comment:
Surely this code shouldn't even run if `dpiSetting` doesn't exist? The feature depends on it, no?

Reply:
Yeah, but I didn't have a check for that and was being defensive.  I put the check in and took out all the defensive stuff.